README.md

```diff
@@ -1,20 +1,25 @@
 This PHP library implements a subscriber for PubSubHubbub.
 
-It was written by Josh Fraser (joshfraser.com) and is released under the Apache 2.0 License
+It was written by [Josh Fraser](http://joshfraser.com) and is released under the Apache 2.0 License
 
-# Usage
+# Install
+Update your `composer` require block:
+```json
+"require": { "pubsubhubub/subscriber": "*" }
+```
```

Comment:
I will suggest to make use of `composer require pubsubhubub/subscriber` than editing composer.json